### ISSUE TITLE
Add test for the FlowCrypt for Thunderbird implementation

### ISSUE THREAD

**Issue by martgil** [FlowCrypt/flowcrypt-browser#5786]
Description: 

In my investigation, there's not much option for testing Thunderbird mail extensions within the Thunderbird Mail Client. So to make our progress in Thunderbird mail development reliable, test must be performed and emulated in puppeteer.

Reference: https://www.zenrows.com/blog/puppeteer-user-agent

**Comment by martgil** [comment]
cc: @sosnovsky

**Comment by sosnovsky** [comment]
Hmm, I'm not sure that changing user agent will make it possible to test Thunderbird implementation, as browser environment will still miss some properties and methods specific for thunderbird (like `browser.composeAction`, `browser.messageDisplayAction` etc)

**Comment by martgil** [comment]
Yeah, you are right @sosnovsky. Oh my. It's not easy to add test for Thunderbird.
